src/model.ts

```diff
@@ -5,6 +5,13 @@ export interface ValidationResult {
   message: string;
 }
 
+// Since is an interface no way to use instanceof
+// Using type guard
+export const isSyncValidationResult = (
```

Comment:
Move to helper?

Reply:
but this helper is not generic, should we move to something like model.helper?

Reply:
Do we need create a generic method? I mean, we have `FieldValidationFunctionSyncAsync` and `RecordValidationFunctionSyncAsync` too

Reply:
Not sure, this is used in two places in the validation-engine if you could add more explanation examples that would be great to understand the update you want to make